src/jit/compiler.hpp

```diff
@@ -2843,7 +2843,22 @@ bool                Compiler::fgIsThrowHlpBlk(BasicBlock * block)
     if (!(block->bbFlags & BBF_INTERNAL) || block->bbJumpKind != BBJ_THROW)
         return false;
 
-    GenTreePtr  call = block->bbTreeList->gtStmt.gtStmtExpr;
+    GenTree* call = nullptr;
+    if (block->IsLIR())
+    {
+        // TODO(pdg): it would be nice if there was simply a bit on the block we could check.
+        for (GenTree* node : LIR::AsRange(block))
+        {
+            if (node->OperGet() == GT_CALL)
+            {
+                call = node;
+            }
+        }
+    }
```

Comment:
The call we are looking for will be a void, top-level call. We could presumably just look at the last node in the range. I'm not sure where this func is used. Worst case, we should reverse iterate from the end of the block.